src/buildVariables.js

```diff
@@ -49,9 +50,7 @@ const buildGetListVariables = introspectionResults => (
     }
 
     if (params.sort) {
-        result['order_by'] = {
-            [params.sort.field]: params.sort.order.toLowerCase(),
-        };
+        result['order_by'] = dotNotationToObject(params.sort.field, params.sort.order.toLowerCase());
```

Comment:
please, check out `_.set` which imo does what you are doing with `dotNotationToObject` already

result['order_by'] = _.set({}, params.sort.field, params.sort.order.toLowerCase())